subsys/settings/src/settings_line.c

```diff
@@ -188,13 +153,8 @@ int settings_line_len_calc(const char *name, size_t val_len)
 {
 	int len;
 
-#ifdef CONFIG_SETTINGS_USE_BASE64
-	/* <enc(value)> */
-	len = val_len/3*4 + ((val_len%3) ? 4 : 0);
-#else
 	/* <evalue> */
 	len = val_len;
-#endif
 	/* <name>=<enc(value)> */
 	len += strlen(name) + 1;
```

Comment:
Would it be possible to move `settings_line_len_calc` to `settings_priv.h` as an inline function? This has basically become one line function now.

Reply:
Added a commit to make this change.

Reply:
Thanks, would you agree on doing similar change to `settings_line_val_get_len` and `settings_line_val_read`? Same reasoning?

Reply:
I'm not sure how easy it will be tot move `settings_line_val_get_len` because of its reference to `settings_io_cb`.   Made the change for settings_line_val_read